content/learn/migration-guides/0.11-0.12/_index.md

```diff
@@ -192,7 +192,17 @@ Also, as you may have noticed, the set of events has changed. The most important
 
 #### `UntypedHandle`
 
-`HandleUntyped` has been renamed to `UntypedHandle`.
+`HandleUntyped` and `HandleId` have been renamed to `UntypedHandle` and `UntypedAssetId`.

```

Comment:
One last nitpick:

I would put "`HandleUntyped` renamed to `UntypedHandle`" on its own line, and then on the next line put:
"`HandleId` has been replaced with `UntypedAssetId` and its typed equivalent `AssetId<T>`."

The justification for saying "replaced" is that it was a bit more than just a rename since type information is now stored in the id (whereas before it was stored in the handle).

Reply:
Good point. Sec.

Reply:
There ya go :)